app-vcs/git-credential-oauth/autobuild/prepare

```diff
@@ -0,0 +1,20 @@
+abinfo "Setting CGO_*FLAGS from environment conterparts ..."
+
+export CGO_CPPFLAGS="${CPPFLAGS}"
+export CGO_CFLAGS="${CFLAGS}"
+export CGO_CXXFLAGS="${CXXFLAGS}"
+export CGO_LDFLAGS="${LDFLAGS}"
+
+abinfo "Setting GOFLAGS for a reproducible build ..."
+export GOFLAGS="${GOFLAGS} -trimpath"
+
+abinfo "Setting other GOFLAGS ..."
```

Comment:
Why? These are already set by Autobuild.

Reply:
Actually this comes from <https://github.com/AOSC-Dev/aosc-os-abbs/blob/a57044af65d1097af5e7abd55116a47139637a1a/app-proxy/gost/autobuild/prepare>, which is also a pure-Go executable with no extra dependencies.